frontend/src/app/modules/external-booking/external-booking-choose-meeting-page/external-booking-choose-meeting-page.component.sass

```diff
@@ -47,3 +47,6 @@ h1
 ::ng-deep .card.mat-card:hover
     box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25) !important
     cursor: pointer
+
+.meeting-cards
+    min-height: 200px
```

Comment:
It would be better to have bigger min-height
and cannot we use %?

Reply:
We cannot use %

Reply:
I think it`s convenient min-height. 
![image](https://user-images.githubusercontent.com/71783287/189682387-5eacf84b-eca8-4213-9a28-ac24cb66f138.png)

Reply:
why we cannot use %?